package.json

```diff
@@ -48,7 +48,7 @@
     "@dimensiondev/kit": "0.0.0-20210115033306-5d4304b",
     "@dimensiondev/matrix-js-sdk-type": "=1.0.0-20200731085604-dc2de92",
     "@dimensiondev/metamask-extension-provider": "^3.0.0-20210125122006-86e9bae",
-    "@dimensiondev/stego-js": "=0.11.1-20201027083223-8ab41be",
+    "@dimensiondev/stego-js": "=0.13.0-20201224101931-64ee02e",
```

Comment:
Hi plz rebase to master and add this deps in packages/maskbook/package.json thanks